match/bin/match

```diff
@@ -88,7 +88,7 @@ class MatchApplication
       c.action do |args, options|
         params = FastlaneCore::Configuration.create(Match::Options.available_options, options.__hash__)
         params.load_configuration_file("Matchfile")
-        decrypted_repo = Match::GitHelper.clone(params[:git_url], params[:shallow_clone])
+        decrypted_repo = Match::GitHelper.clone(params[:git_url], params[:shallow_clone], params[:git_branch])
```

Comment:
That's what's causing the merge conflict, we moved the code to `lib/commands_generator.rb`